File: RssInfo.py
# encoding: utf-8
# Rodrigo Fuentes Hernandez
# aka Blurred_9L
# Compiladores CUCEI 2013B
# RssInfo.py

class RssInfo( object ):
    
    def __init__( self ):
        self.channels = list()
        self.inItem = False
        self.errorsFound = 0

File: RssInfo.rb
# encoding: utf-8
# Rodrigo Fuentes Hernandez
# aka Blurred_9L
# Compiladores CUCEI 2013B
# RssInfo.rb

require "singleton"

class RssInfo
    attr_reader :channels, :inItem, :errorsFound
    attr_writer :channels, :inItem, :errorsFound
    
    include Singleton
    
    public
        def initialize()
            @channels = Array.new()
            @inItem = false
            @errorsFound = 0
        end
end
